src/server/qgsserverprojectutils.h

```diff
@@ -154,6 +154,14 @@ namespace QgsServerProjectUtils
    */
   SERVER_EXPORT int wmsMaxAtlasFeatures( const QgsProject &project );
 
+  /**
+   * Returns the denominator of the default scale used in case of the scale is not given
+   * \param project the QGIS project
+   * \returns the denominator of the scale
```

Comment:
Is it the denominator or map units per mm?

For metric systems, a denominator 1000 (1:1000) would be 1m/mm, so 1 map unit per mm.

Reply:
Comment is wrong.
It's about map units per mm. 
The Scaledenominator is converted at gui level depending on it's unittype into map units per mm.
Will change that. Thanks.